src/components/card/CardSidebar.vue

```diff
@@ -389,7 +392,8 @@ export default {
 			}
 
 			this.copiedCard = JSON.parse(JSON.stringify(this.currentCard))
-			this.allLabels = this.currentCard.labels
+			// this.allLabels = this.currentCard.labels
+			this.allLabels = [...this.currentCard.labels].sort((a, b) => (a.title < b.title) ? -1 : 1)
```

Comment:
I think the naming is now a bit confusing, allLabels should rather be assignedLabels.